content/en/_index.md

```diff
@@ -197,6 +242,39 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="Node.js" >}}
+// Imports the Google Cloud Data Loss Prevention library
+const DLP = require('@google-cloud/dlp');
```

Comment:
Sounds like we landed on something different, I'd make sure we update to: 
```
const {DlpServiceClient} = require('@google-cloud/dlp').v2;




  // Instantiates a client

  const dlpClientInstance = new DlpServiceClient();
```

And it should probably be changed in line 346 as well.

Reply:
Done.